src/main/java/de/tum/cit/aet/artemis/core/web/AccountResource.java

```diff
@@ -133,46 +139,87 @@ public ResponseEntity<Void> changePassword(@RequestBody PasswordChangeDTO passwo
     }
 
     /**
-     * PUT account/ssh-public-key : sets the ssh public key
+     * GET account/ssh-public-keys : retrieves all SSH keys of a user
      *
-     * @param sshPublicKey the ssh public key to set
+     * @return the ResponseEntity containing all public SSH keys of a user with status 200 (OK)
+     */
+    @GetMapping("account/ssh-public-keys")
```

Comment:
I suggest to move all REST calls related to ssh public keys into its own resource